mhr_api/report-templates/template-parts/search-result-ppr/amendmentStatement.html

```diff
@@ -58,8 +58,6 @@
 </div>
 {% endif %}
 
-<div class="container pt-4">
-  [[search-result-ppr/registeringParty.html]]
+<div class="container pt-4 pb-4">
+    [[search-result-ppr/registeringParty.html]]
```

Comment:
Intentional extra spacing?

Reply:
Yes, the UXA feedback was to remove the separator line but retain the spacing.

Reply:
kk right on!